docs/installation.md

```diff
@@ -36,11 +36,34 @@ Now in the `~/govuk` directory, run the following commands.
 git clone git@github.com:alphagov/govuk-docker.git
 cd govuk-docker
 bundle install
-govuk-docker setup
+bundle exec bin/setup
```

Comment:
Can we avoid the need to run these bins inside `bundle exec`?

As I understand from https://bundler.io/guides/bundler_setup.html we can add:

```
require 'rubygems'
require 'bundler/setup'
```

for that to work.